### PR TITLE
ability to change 'fillShadowGradient' color and 'fillShadowGradient'…

### PR DESCRIPTION
**Pull request by Aleksandern**
Ability to change 'fillShadowGradient' color and 'fillShadowGradient' opacity.
It's needed for 'BarChart'

### CODE REVIEW

**Comment by Aleksandern** [comment]
@Hermanya 
Review please.

File: src/abstract-chart.js [hunk 1]
```diff
@@ -258,12 +266,12 @@ class AbstractChart extends Component {
         >
           <Stop
             offset="0"
-            stopColor={this.props.chartConfig.color()}
-            stopOpacity="0.1"
+            stopColor={fillShadowGradient}
+            stopOpacity={fillShadowGradientOpacity}
```
Comment (Hermanya) [inline src/abstract-chart.js]: it's kind of odd that you can set the first Stop opacity but not the second one's

**Comment by Hermanya** [comment]
This is now available in v3.11.0, thank you for your contribution 👍

**Comment by futantan** [comment]
hi~ I'm trying to change the color of the shadow below lines. 
image
How can I change the shadow gradient from for example opacity 1 to 0.1 like above image?

**Comment by Hermanya** [comment]
Right now it goes from fillShadowGradientOpacity to 0. PR to change this is welcome, read my comments above

**Comment by futantan** [comment]
I will create a PR a few days later.
How about migrating to TypeScript, if that's acceptable, I'm considering to create a PR too

**Comment by Hermanya** [comment]
Thatd be interesting. However heads up I'm going to be unavailable for a week. Today my vacation begins.

**Comment by futantan** [comment]
That's fine. Have a nice vacation~

**Pull request merged**